### PR TITLE
refactor(contributing.md): made all code-commands into shell snippets…

### PR DESCRIPTION
**Pull request by TheMcnafaha**
…, easier to copy and paste

Shell snippets can be copied with the press of a button and are already present throughout the file. This change makes them present any  time a piece of code is suggested.

re #476

# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [x] Docs / tests

# Description

Made all code-suggestions easier to copy and paste. Made file more uniform in style (all shell snippets now).

# Use cases and why
1. More effective guide

# Checklist:

- [x] My code follows the [developer guidelines of this project](https://github.com/qwikifiers/qwik-ui/blob/main/CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

**Comment by TheMcnafaha** [comment]
I have read the CLA Document and I hereby sign the CLA

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -58,17 +60,29 @@ git checkout -b my-fix-branch main
...
+```
 
 - Visit the URL printed in the console and you'll have a page opened with the suite of widgets.
 
 - Once you made some changes in either package (`headless`, `tailwind` or `material`), you will see them immediately reflected on the page.
 
 ### ▶ 5. Make sure you add / modify tests
 
-Run `pnpm run test` to make sure there aren't any errors
+Run the follwoing command to make sure there aren't any errors:
```
Comment (maiieul) [inline CONTRIBUTING.md:81]: small typo on "following" here

**Review by maiieul — CHANGES_REQUESTED** [review]

**Review by gioboa — APPROVED** [review]
Thanks @TheMcnafaha 👏

**Comment by gioboa** [comment]
@all-contributors please add @TheMcnafaha for code, ideas, maintenance

**Pull request merged**